src/components.ts

```diff
@@ -312,7 +351,14 @@ export abstract class Component<T extends object = object> extends Controller<T>
     return (viewName) ?
       this.app.views.find(viewName, contextView.namespace) : contextView;
   }
-
+  /**
+   * Retrieve the appropriate view attribute's value for a given view instance
```

Comment:
```suggestion
   * Retrieve the appropriate view attribute's value for a given view instance.
   * If the value is a template, it will be rendered prior to being returned.
```